application/controllers/api/Key.php

```diff
@@ -233,16 +233,16 @@ private function _generate_key()
     private function _get_key($key)
     {
         return $this->db
-			->where(config_item('rest_key_column'), $key)
-			->get(config_item('rest_keys_table'))
-			->row();
+            ->where(config_item('rest_key_column'), $key)
+            ->get(config_item('rest_keys_table'))
+            ->row();
     }
 
     private function _key_exists($key)
     {
         return $this->db
-			->where(config_item('rest_key_column'), $key)
-			->count_all_results(config_item('rest_keys_table')) > 0;
+            ->where(config_item('rest_key_column'), $key)
+            ->count_all_results(config_item('rest_keys_table')) > 0;
```

Comment:
What IDE do you use? It's not a problem, just curious. I use a mixture of Netbeans PHP and SublimeText for web development.